rest/route/middleware.go

```diff
@@ -48,7 +48,17 @@ func (m *projCtxMiddleware) ServeHTTP(rw http.ResponseWriter, r *http.Request, n
 
 	user := gimlet.GetUser(ctx)
 
-	if opCtx.ProjectRef != nil && opCtx.ProjectRef.Private && user == nil {
+	pref, err := opCtx.GetProjectRef()
+	if err != nil {
```

Comment:
Should probably handle if pref is nil

Reply:
I don't think it can ever be nil, except (perhaps) if there are no projects in evergreen, which we should figure out how to bootstrap/migrate our way out of